src/azul/deployment.py

```diff
@@ -145,6 +145,10 @@ def securityhub(self):
     def sns(self):
         return self.client('sns')
 
+    @property
+    def ses(self):
```

Comment:
```suggestion
    def ses(self) -> …:
```

Using the appropriate client stub from boto3-stubs